csrankings-d.csv

```diff
@@ -1335,7 +1335,7 @@ Dong-Guk Shin,University of Connecticut,http://profiles.uconn.edu/display/64589,
 Dong-Ming Yan 0001,Chinese Academy of Sciences,https://sites.google.com/site/yandongming,xAFSO3AAAAAJ
 Dong-Sun Kim 0001,Kyungpook National University,http://www.darkrsw.net,Rmzs0QIAAAAJ
 Dongbing Gu,University of Essex,https://www.essex.ac.uk/people/gudon81301/dongbing-gu,U-3knAUAAAAJ
-Dongfang Liu,Rochester Institute of Technology,https://www.rit.edu/directory/dxleec-dongfang-liu,uICY0vEAAAAJ
+Dongfang Liu,Rochester Institute of Technology,https://dongfang-liu.github.io,uICY0vEAAAAJ

```

Comment:
CE, not CS. Please explain how this faculty meets the inclusion criteria.

Reply:
Although Dongfang Liu is in the department of CE, he is a full-time tenure-track faculty member and advises Ph.D. students in Computer Science.

Reply:
Please provide evidence he can _solely_ advise PhD students for a PhD in Computer Science. Thanks.